src/test/osd/osd-scrub-repair-auto-repair.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+source $(dirname $0)/../detect-build-env-vars.sh
```

Comment:
you don't need to have a dedicated sh script for wrapping `src/test/osd/osd-scrub-repair.sh`.

what you need is

```cmake
add_ceph_test(osd-scrub-repair.sh-jerasure
  ${CMAKE_CURRENT_SOURCE_DIR}/osd-scrub-repair.sh
    TEST_auto_repair_erasure_coded_appends
    TEST_auto_repair_erasure_coded_overwrites)
```